pkg/daemon/ceph/client/mgr_test.go

```diff
@@ -0,0 +1,42 @@
+package client
+
+import (
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/rook/rook/pkg/clusterd"
+	exectest "github.com/rook/rook/pkg/util/exec/test"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestEnableModule(t *testing.T) {
+	executor := &exectest.MockExecutor{}
+	executor.MockExecuteCommandWithOutputFile = func(debug bool, actionName, command, outputFile string, args ...string) (string, error) {
+		logger.Infof("Command: %s %v", command, args)
+		switch {
+		case args[0] == "mgr" && args[1] == "module" && args[2] == "enable":
+			if args[3] == "prometheus" || args[3] == "pg_autoscaler" || args[3] == "crash" {
+				return "", nil
+			}
+
+		case args[0] == "mgr" && args[1] == "module" && args[2] == "disable":
+			if args[3] == "prometheus" || args[3] == "pg_autoscaler" || args[3] == "crash" {
+				return "", nil
+			}
+		}
+
+		return "", errors.Errorf("unexpected ceph command %q", args)
+	}
+
+	err := enableModule(&clusterd.Context{Executor: executor}, "clusterName", "pg_autoscaler", true, "enable")
+	assert.Nil(t, err)
```

Comment:
Use `assert.Error` and `assert.NoError` for testing error results from tests, please.

Reply:
is there any advantage of using one over the other?

Reply:
They are equivalent, but `assert.NoError()` tends to be more readable than `assert.Nil()`